code/modules/language/language_holder.dm

```diff
@@ -272,6 +272,11 @@ Key procs
 /datum/language_holder/lizard/ash
 	selected_language = /datum/language/draconic
 
+/datum/language_holder/dwarf
+	understood_languages = list(/datum/language/dwarven = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/dwarven= list(LANGUAGE_ATOM))
+	blocked_languages = list(/datum/language/common = list(LANGUAGE_ATOM))
```

Comment:
blocked? how in sam hell are they supposed to trade?

Reply:
the book that dwarves start with lets one of em speak common iirc

Reply:
Actually it might not. Hmm.

Reply:
Remove the book. Curators can speak any language. That can work for translation.

Reply:
But goof that's a terrible idea

Reply:
It worked for my lava dwarves. What's terrible about adding a purpose to the curator? Explain your statements.

Reply:
And when you say add a purpose you mean make it incredibly inconvenient for people to operate without the Curator, despite the fact that the Curator doesn't even have mining access.

Reply:
Bad idea goof, curator can help by talking to all the dwarves as will if he so wishes. If he exists.

Reply:
This is a bit of a failing of mine when I redid language holders, I'll see if I can't do this smarter. Leave this for now.